src/screens/authentication/utils/index.js

```diff
@@ -0,0 +1 @@
+export const isNotEmail = (text) => !text.includes('@')
```

Comment:
A very small detail, but I had to come here to see what `isNotEmail` was. I'd suggest renaming to `isValidEmail`. @nahudalla what do you say?

Reply:
Yes, I agree. Also, this is not a correct way of checking if it's not a valid email address... but I think it's OK for now

Reply:
You are correct, I considered if putting a RegExp was necessary as a badly formatted email would not be logged by firebase and only needed the distinction between username and email. 

I will rethink if what I intended is a "**isUsername**" functionality instead.